operator/src/main/java/org/keycloak/operator/crds/v2alpha1/deployment/KeycloakSpec.java

```diff
@@ -360,4 +365,12 @@ public void setReadinessProbeSpec(ProbeSpec readinessProbeSpec) {
     public void setStartupProbeSpec(ProbeSpec startupProbeSpec) {
         this.startupProbeSpec = startupProbeSpec;
     }
+
+    public SchedulingSpec getImportSchedulingSpec() {
```

Comment:
Does it make sense for us to take this opportunity to nest the import scheduling so we have `spec.import.scheduling`?

The benefits would be:

1. Consistent with `spec.update.scheduling`
2. Allows for future global import configuration that maybe required e.g. `spec.update.<tbdElement>`. I don't have a concrete example, but you never know what the future might bring.

Reply:
> Does it make sense for us to take this opportunity to nest the import scheduling so we have spec.import.scheduling?

It does, we talked about this a little in our team meeting, but I didn't reflect it in this pr. It's possible that users will eventually want to tweak the resource allocation for the update and import jobs as well, so having a separate stanza is good.